src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/validation/FormAValidatorTest.java

```diff
@@ -220,7 +219,7 @@ public void shouldFailFieldsHavingInvalidValues() {
 
         assertThat(validationResult.getStatus(), is(WARNINGS));
         assertThat(validationResult.getErrors(), is(emptyList()));
-        assertThat(validationResult.getWarnings().size(), is(24));
+        assertThat(validationResult.getWarnings().size(), is(23));
```

Comment:
I question the utility of this line.
Happy to keep, but if you agree this doesn't add value, I'm happy to support its removal.

Reply:
well, it caught the fact I have changed logic, so it actually adds little value imho :)